pandas/tests/arithmetic/test_datetime64.py

```diff
@@ -2137,7 +2137,12 @@ def test_timedelta64_equal_timedelta_supported_ops(self, op):
         #                   'us': 1}
 
         def timedelta64(*args):
-            return sum(starmap(np.timedelta64, zip(args, intervals)))
+            # integer types -> m8 casting will soon be prohibited
```

Comment:
can you remove most of this comment, and just add a ref to the relevant numpy issue.

Reply:
Done